README.md

```diff
@@ -187,16 +187,16 @@ fastify.register(jwt, {
   // Global default signing method options
   sign: {
     algorithm: 'ES256',
-    issuer: 'api.example.tld'
+    isss: 'api.example.tld'
```

Comment:
```suggestion
    iss: 'api.example.tld'
```

Reply:
Whoops held my finger too long on the keyboard 😸